clang-tools-extra/docs/clang-tidy/checks/misc/must-use.rst

```diff
@@ -0,0 +1,28 @@
+.. title:: clang-tidy - misc-must-use
+
+misc-must-use
+=============
+
+Allows for strictly enforce variables are used for specific classes, even with
```

Comment:
Please make this statement same as in Release Notes.

Reply:
Done.